internal/runner/runner.go

```diff
@@ -465,80 +533,41 @@ func (r *Runner) Close() {
 	r.hm.Close()
 }
 
-// TODO - wip - just ignore
 func (r *Runner) wildcardWorker() {
 	defer r.wgwildcardworker.Done()
 
-	<-r.wildcardchan
-
-	if r.hm == nil {
-		return
-	}
-
-	wildcards := make(map[string]struct{})
-	ipDomain := make(map[string]map[string]struct{})
-
-	// prepare in memory structure similarly to shuffledns
-	r.hm.Scan(func(k, v []byte) error {
-		var dnsdata retryabledns.DNSData
-		err := dnsdata.Unmarshal(v)
-		if err != nil {
-			// the item has no record - ignore
-			return nil
-		}
-
-		for _, a := range dnsdata.A {
-			_, ok := ipDomain[a]
-			if !ok {
-				ipDomain[a] = make(map[string]struct{})
-			}
-			ipDomain[a][string(k)] = struct{}{}
-		}
-
-		return nil
-	})
-
 	// process all items
-	for A, hosts := range ipDomain {
-		// We've stumbled upon a wildcard, just ignore it.
-		if _, ok := wildcards[A]; ok {
-			continue
-		}
+	for {
+		wildcardCheck, more := <-r.wildcardworkerchan
+		if more {
```

Comment:
I would reverse the condition with:
```
if !more {
   break
}
hosts := wildcardCheck.Hosts
...
```

Reply:
I've changed it like you asked, please take a look again.